.github/workflows/update-visual-tests.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   update-visual-tests:
     runs-on: ubuntu-latest
     container:
-      image: mcr.microsoft.com/playwright:v1.44.1-jammy
+      image: mcr.microsoft.com/playwright:v1.48.2-jammy
```

Comment:
I've changed the PW version based on this error log.
https://github.com/kiwicom/orbit/actions/runs/11632651461/job/32396199861?pr=4497

Reply:
Also, I had to update visual tests as they were failing.